### ISSUE TITLE
Review -> Discrepancies between Initial and Review aren't showing

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#507]
This is not showing that there is a difference between the initial and the review (i.e., no number like it shows for associations). Possibly because the review is just not checking one of the zones that the initial person did?
![CoRA_Discrepancies not showing](https://user-images.githubusercontent.com/7350474/63068036-46685700-bed7-11e9-965c-09f61901474d.png)

**Comment by clegarde** [comment]
Some more info for this issue. It seems that Measurements and General are marking discrepancies correctly. The Associations one is marking "3" regardless of what has been entered. In the example below, all fields are blank from the Initial and Review but after clicking "save" it still shows 3.
So - General and Measurements might be working; Zones and Associations are not; not sure about the rest.
![CoRA_discrepancies](https://user-images.githubusercontent.com/7350474/63452259-751c8b00-c40b-11e9-9ade-4510eca19a40.png)

**Comment by SachinPawaskarUNO** [comment]
@clegarde Please verify

**Comment by clegarde** [comment]
I didn't test one with limited zones. This is in the USS Oklahoma Project. I would think this should show as 2 discrepancies? Not sure why it says 10.
![Screen Shot 2019-10-27 at 9 05 29 PM](https://user-images.githubusercontent.com/7350474/67646879-12110e00-f8fe-11e9-8a4f-bd29f4e484e2.png)

**Comment by SachinPawaskarUNO** [comment]
@clegarde I agree that should say 2 discrepancies, We will take a look at this.
